src/javascript/expression.rs

```diff
@@ -147,6 +195,7 @@ impl<'module> Generator<'module> {
     }
 
     fn call<'a>(&mut self, fun: &'a TypedExpr, arguments: &'a [CallArg<TypedExpr>]) -> Output<'a> {
+        // println!("ioooo {:?}", fun);
```

Comment:
This needs to short circuit printing, i.e. If directly calling a record constructor use literal construction.